### ISSUE TITLE
Ansible Galaxy imports failing

### ISSUE THREAD

**Issue by markgoddard** [michaelrigart/ansible-role-interfaces#143]
We have an action to automatically import this role, but it seems to be failing: https://github.com/michaelrigart/ansible-role-interfaces/actions/workflows/publish-role.yml

**Comment by cityofships** [comment]
Is this because `secrets.GALAXY_API_KEY` is missing?

**Comment by markgoddard** [comment]
Yes, I expect so. @michaelrigart , are you able to set this secret for the repo?

**Comment by michaelrigart** [comment]
@markgoddard : I just set the Ansible API key as repo secret, but the action still seems to fail.

**Comment by cityofships** [comment]
> @markgoddard : I just set the Ansible API key as repo secret, but the action still seems to fail.
> 
> Could this be due to the capitalization of my Galaxy username MichaelRigart vs small-caps on github (michaelrigart), since the command it runs is: ansible-galaxy import michaelrigart xyz?

Yes, that's the reason - gives 500 if tried from command line too.

**Comment by michaelrigart** [comment]
It seems that I can't change the namespace on Galaxy neither.

**Comment by michaelrigart** [comment]
I have created an issue on the Galaxy GH repo. Hopefully it can be resolved: https://github.com/ansible/galaxy/issues/3217

**Comment by markgoddard** [comment]
Since the roll out of Galaxy NG there is no UI for importing roles, and when I tried to do it manually it took a few goes to make it work.

First, you need to use the capitalised MichaelRigart namespace. Next, you need to specify the tag via `--branch`, otherwise it uploads the `master` branch.

```
ansible-galaxy role import MichaelRigart ansible-role-interfaces  --api-key <key> --branch v1.14.4
```

I've updated the GitHub actions publish workflow to do this, but unfortunately it failed with a 403. @michaelrigart perhaps you need to regenerate a new API token for NG galaxy?
